static/styles/zulip.css

```diff
@@ -435,7 +435,10 @@ li,
     outline-color: hsl(0, 0%, 73%);
     height: 18px;
     width: 10px;
-    padding: 6px 6px;
+    padding-top: 8px;
+    padding-right: 3px;
+    padding-left: 13px;
+    padding-bottom: 0.5px;
```

Comment:
You may want to condense it into a single `padding` property to follow the convention.

Reply:
@aryanshridhar yep will do that .